Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.h

```diff
@@ -18,34 +17,22 @@ namespace ShaderManagementConsole
 {
     class ShaderManagementConsoleApplication
         : public AtomToolsFramework::AtomToolsApplication
-        , private ShaderManagementConsoleWindowNotificationBus::Handler
     {
     public:
         AZ_TYPE_INFO(ShaderManagementConsole::ShaderManagementConsoleApplication, "{A31B1AEB-4DA3-49CD-884A-CC998FF7546F}");
 
         using Base = AtomToolsFramework::AtomToolsApplication;
 
         ShaderManagementConsoleApplication(int* argc, char*** argv);
-        virtual ~ShaderManagementConsoleApplication() = default;
+        virtual ~ShaderManagementConsoleApplication();
 
         //////////////////////////////////////////////////////////////////////////
         // AzFramework::Application
         void CreateStaticModules(AZStd::vector<AZ::Module*>& outModules) override;
         const char* GetCurrentConfigurationName() const override;
 
     private:
-        //////////////////////////////////////////////////////////////////////////
-        // ShaderManagementConsoleWindowNotificationBus::Handler overrides...
-        void OnShaderManagementConsoleWindowClosing() override;
-        //////////////////////////////////////////////////////////////////////////
-
-        //////////////////////////////////////////////////////////////////////////
-        // AzFramework::Application overrides...
-        void Destroy() override;
-        //////////////////////////////////////////////////////////////////////////
-
         void ProcessCommandLine();
```

Comment:
You'll need to merge changes from my other PR to fix this and make it consistent with the base class and the MaterialEditor